sdk/src/Core/Amazon.Runtime/Internal/Auth/BearerTokenSigner.cs

```diff
@@ -55,7 +52,7 @@ private void InternalSign(IRequest request, BaseIdentity baseIdentity)
 
             if (baseIdentity is not AWSToken awsToken || string.IsNullOrEmpty(awsToken.Token))
             {
-                throw new AmazonClientException("No Token found.  Operation requires a Bearer token.");
+                throw new AmazonClientException("No Token found. Operation requires a Bearer token.");
```

Comment:
This is annoying, but the bearer token signer tests assert the message with the extra space...

https://github.com/aws/aws-sdk-net/blob/f2b2126357d7095d72c3cf889bb3c6411db788a9/sdk/test/UnitTests/Custom/Runtime/BearerTokenSignerTests.cs#L255

https://github.com/aws/aws-sdk-net/blob/f2b2126357d7095d72c3cf889bb3c6411db788a9/sdk/test/UnitTests/Custom/Runtime/BearerTokenSignerTests.cs#L285

Reply:
I updated the test to not have the extra space. The space just bugs me.